src/main/java/racingcar/racing/view/InputView.java

```diff
@@ -0,0 +1,25 @@
+package racingcar.racing.view;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class InputView {
+    private static final Scanner scanner = new Scanner(System.in);
```

Comment:
상수이므로 대문자로 작성해주셔야 합니다 🙏 
```suggestion
    private static final Scanner SCANNER = new Scanner(System.in);
```
`jetbrains` 컨벤션엔 없을 수도 있는데 사실 상수 대문자 컨벤션은 `Java` 기본 컨벤션이라서 
지켜주시면 좋을 것 같습니다 😄 
> https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names